### ISSUE TITLE
Can this work on the Efergy Hub as well? 

### ISSUE THREAD

**Issue by ttenterprises** [hardtechnology/EfergyMQTT#13]
Hi Love your work I'm thinking of buying several transmitters and an efergy hub.
I would like to incorporate your application into the hub to push the data into my home automation system .
Do you think it will work with the hub and multiple transmitters?
cheers
Tom

**Comment by hardtechnology** [comment]
Hi, This is a replacement fot the Efergy hub, you use one of your existing receivers and add this to it to provide MQTT capability. I actually haven't used a Hub to know what functionality it could provide.

**Comment by buzzware** [comment]
Yes, but if you're building your own human interface instead of the supported ones, the hub makes sense. Does anyone know if it works? Alternatively, can you get the 433mhz signal directly with an appropriate receiver chip?
